Adds emergency shutters, cameras, and disposals

BillyBangles opened the pull request:
As well as some other minor map fixes.

OrbisAnima commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -413,3 +413,98 @@
 	item_state = "mbill"
 
 
```
move all this new code to 
code\modules\eros\clothing\miscellaneous.dm

Dinnnertime committed:
Merge remote-tracking branch 'refs/remotes/origin/dev' into unzipsformatting

Dinnnertime committed:
Name notification bugfix

Will no longer ping the AI at any mention of the letters "ai" in any
word (for example, "wait")

Dinnnertime committed:
Merge remote-tracking branch 'refs/remotes/ErosResearchPlatform/dev' into unzipsformatting

Dinnnertime committed:
Merge remote-tracking branch 'refs/remotes/origin/unzipsformatting' into dev

silveryferret committed:
Changes Soma description, removes it from random medical spots.

silveryferret committed:
Let's mentors see/use faxes

Dinnnertime committed:
Merge remote-tracking branch 'refs/remotes/ErosResearchPlatform/dev' into dev

Dinnnertime committed:
Merge remote-tracking branch 'refs/remotes/ErosResearchPlatform/newerosmap' into somebodyswatchingme

Dinnnertime committed:
Merge remote-tracking branch 'refs/remotes/ErosResearchPlatform/newerosmap' into somebodyswatchingme

Dinnnertime committed:
Merge remote-tracking branch 'refs/remotes/ErosResearchPlatform/newerosmap' into somebodyswatchingme

Dinnnertime committed:
finishes cargo

all the items in cargo didn't get added for some reason. this puts them
back in.

Dinnnertime committed:
adds a bunch of missing emergency shutters

as well as fixing a few map oddities

Dinnnertime committed:
Merge remote-tracking branch 'refs/remotes/ErosResearchPlatform/newerosmap' into somebodyswatchingme

Dinnnertime committed:
Merge remote-tracking branch 'refs/remotes/ErosResearchPlatform/newerosmap' into somebodyswatchingme

Dinnnertime committed:
Merge remote-tracking branch 'refs/remotes/ErosResearchPlatform/newerosmap' into somebodyswatchingme

Dinnnertime committed:
Adds Eros and Cargo camera networks

Makes the Eros into the default camera network. Also begins adding
cameras to the map.
